Lib/test/test_pyexpat.py

```diff
@@ -771,6 +771,48 @@ def resolve_entity(context, base, system_id, public_id):
         self.assertEqual(handler_call_args, [("bar", "baz")])
 
 
+class ParentParserLifetimeTest(unittest.TestCase):
+    """
+    Subparsers make use of their parent XML_Parser inside of Expat.
+    As a result, parent parsers need to outlive subparsers.
+
+    See https://github.com/python/cpython/issues/139400.
+    """
+
+    def test_parent_parser_outlives_its_subparsers__single(self):
+        parser = expat.ParserCreate()
+        subparser = parser.ExternalEntityParserCreate(None)
+
+        # Now try to cause garbage collection of the parent parser
+        # while it's still being referenced by a related subparser
```

Comment:
```suggestion
        # Now try to cause garbage collection of the parent parser
        # while it's still being referenced by a related subparser.
```

I think you can add this in the description of the class, saying that each test will try to delete the parent before the children.